### DISCUSSION TITLE
About provided data

### DISCUSSION THREAD

**Discussion by maho-sk** [Ignitarium-Renesas/RZV2L_AiLibrary#2] [General]
Thank you for developing the libraries.

I have two questions.

① If I want to use the library for additional learning, can you disclose the dataset and annotated data used for learning when creating the AI ​​library?

② If ① is not disclosed, is there a way to relearn if additional learning is required?
　 Is there basically no other way than to ask the partner?

**Comment by mohitb-ign** [comment]
Hello there,
Thanks for the questions. Here are the answers:

Q1 : If I want to use the library for additional learning, can you disclose the dataset and annotated data used for learning when creating the AI library?
Answer: We have disclosed the datasets used for re-training in readme files of every application. 
For example, please have a look here: https://github.com/Ignitarium-Renesas/RZV2L_AiLibrary/tree/main/01_Head_count#model-details
All the data used for training is open source and freely available on internet.
 
Q2 : If ① is not disclosed, is there a way to relearn if additional learning is required? Is there basically no other way than to ask the partner?
Answer:  As mentioned earlier, datasets used in the pre-trained library are open source. Name of the dataset is clearly mentioned in the readme files. 
If any additional training is required for any application to improve the accuracy, user needs to train the model with desired dataset, convert the model to onnx file format.
That onnx file should be translated using DRP- ai translator and then should be used on the board.
